From 69a4dd9114a7a86db01c75029b8be16f118ebc76 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bastien=20ROUCARI=C3=88S?= <roucaries.bastien@gmail.com>
Date: Wed, 14 Dec 2016 15:11:30 +0100
Subject: [PATCH] Remove spurious cache line

This does not allow to apply patch cleanly

find . -path './.git' -prune -o -type 'f' -name '*.html' -exec sed -i 's,^<!-- Magick Cache.*$,,g' {} \;

Forwarded: Yes, https://github.com/ImageMagick/ImageMagick/issues/304
---
 index.html                             | 2 +-
 www/ImageMagickObject.html             | 1 -
 www/advanced-unix-installation.html    | 1 -
 www/advanced-windows-installation.html | 1 -
 www/animate.html                       | 1 -
 www/api.html                           | 1 -
 www/architecture.html                  | 1 -
 www/binary-releases.html               | 1 -
 www/changelog.html                     | 1 -
 www/cipher.html                        | 1 -
 www/color-management.html              | 1 -
 www/color.html                         | 1 -
 www/command-line-options.html          | 1 -
 www/command-line-processing.html       | 1 -
 www/command-line-tools.html            | 1 -
 www/compare.html                       | 1 -
 www/compose.html                       | 1 -
 www/composite.html                     | 1 -
 www/conjure.html                       | 1 -
 www/connected-components.html          | 1 -
 www/convert.html                       | 1 -
 www/display.html                       | 1 -
 www/distribute-pixel-cache.html        | 1 -
 www/download.html                      | 1 -
 www/escape.html                        | 1 -
 www/examples.html                      | 1 -
 www/exception.html                     | 1 -
 www/export.html                        | 1 -
 www/formats.html                       | 1 -
 www/fx.html                            | 1 -
 www/gradient.html                      | 1 -
 www/high-dynamic-range.html            | 1 -
 www/history.html                       | 1 -
 www/identify.html                      | 1 -
 www/import.html                        | 1 -
 www/index.html                         | 1 -
 www/install-source.html                | 1 -
 www/jp2.html                           | 1 -
 www/license.html                       | 1 -
 www/links.html                         | 1 -
 www/magick++.html                      | 1 -
 www/magick-core.html                   | 1 -
 www/magick-script.html                 | 1 -
 www/magick-vector-graphics.html        | 1 -
 www/magick-wand.html                   | 1 -
 www/magick.html                        | 1 -
 www/miff.html                          | 1 -
 www/mogrify.html                       | 1 -
 www/montage.html                       | 1 -
 www/motion-picture.html                | 1 -
 www/opencl.html                        | 1 -
 www/openmp.html                        | 1 -
 www/perl-magick.html                   | 1 -
 www/porting.html                       | 1 -
 www/quantize.html                      | 1 -
 www/resources.html                     | 1 -
 www/security-policy.html               | 1 -
 www/sitemap.html                       | 1 -
 www/stream.html                        | 1 -
 www/support.html                       | 1 -
 www/webp.html                          | 1 -
 61 files changed, 1 insertion(+), 61 deletions(-)

diff --git a/index.html b/index.html
index 247a8ca03..f0f425cff 100644
--- a/index.html
+++ b/index.html
@@ -239,4 +239,4 @@ Use ImageMagick<sup><a href="http://tarr.uspto.gov/servlet/tarr?regser=serial&am
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 10:56 -->
+
diff --git a/www/ImageMagickObject.html b/www/ImageMagickObject.html
index 167058649..8dd468d84 100644
--- a/www/ImageMagickObject.html
+++ b/www/ImageMagickObject.html
@@ -130,4 +130,3 @@ regsvr32 /u /s ImageMagickObject.dll
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 16:06 -->
\ No newline at end of file
diff --git a/www/advanced-unix-installation.html b/www/advanced-unix-installation.html
index b057d655e..186818e00 100644
--- a/www/advanced-unix-installation.html
+++ b/www/advanced-unix-installation.html
@@ -637,4 +637,3 @@ identify -list format
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 10:59 -->
\ No newline at end of file
diff --git a/www/advanced-windows-installation.html b/www/advanced-windows-installation.html
index e713d766e..a287ca568 100644
--- a/www/advanced-windows-installation.html
+++ b/www/advanced-windows-installation.html
@@ -401,4 +401,3 @@ _MAGICKMOD_
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:14 -->
\ No newline at end of file
diff --git a/www/animate.html b/www/animate.html
index bfbf3fd62..222520db9 100644
--- a/www/animate.html
+++ b/www/animate.html
@@ -532,4 +532,3 @@ transparent, extract, background, or shape the alpha channel</td>
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 12:41 -->
\ No newline at end of file
diff --git a/www/api.html b/www/api.html
index 878204730..823cc60fd 100644
--- a/www/api.html
+++ b/www/api.html
@@ -185,4 +185,3 @@
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:00 -->
\ No newline at end of file
diff --git a/www/architecture.html b/www/architecture.html
index e9877ef4f..bfd9518a7 100644
--- a/www/architecture.html
+++ b/www/architecture.html
@@ -1423,4 +1423,3 @@ ModuleExport size_t analyzeImage(Image **images,const int argc,const char **argv
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:01 -->
\ No newline at end of file
diff --git a/www/binary-releases.html b/www/binary-releases.html
index 394fdb1eb..f5f8c56e8 100644
--- a/www/binary-releases.html
+++ b/www/binary-releases.html
@@ -464,4 +464,3 @@ objShell.Exec("magick ""e:/myimages/image.png"" ""e:/myimages/image.jpg""")
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 10:56 -->
\ No newline at end of file
diff --git a/www/changelog.html b/www/changelog.html
index 7172e0057..27bb83eb3 100644
--- a/www/changelog.html
+++ b/www/changelog.html
@@ -288,4 +288,3 @@
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:01 -->
\ No newline at end of file
diff --git a/www/cipher.html b/www/cipher.html
index 3aed26478..80437141f 100644
--- a/www/cipher.html
+++ b/www/cipher.html
@@ -140,4 +140,3 @@ Examples <a href="https://www.imagemagick.org/Usage/transform/#encipher"
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 12:00 -->
\ No newline at end of file
diff --git a/www/color-management.html b/www/color-management.html
index df29e71a7..176baca52 100644
--- a/www/color-management.html
+++ b/www/color-management.html
@@ -150,4 +150,3 @@ convert myimage_channels_*.png -set colorspace HSL -combine -colorspace RGB -set
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:14 -->
\ No newline at end of file
diff --git a/www/color.html b/www/color.html
index 6355e689f..af9ab3257 100644
--- a/www/color.html
+++ b/www/color.html
@@ -4951,4 +4951,3 @@ device-cmyk(0.11, 0.48, 0.83, 0.00)
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 10:56 -->
\ No newline at end of file
diff --git a/www/command-line-options.html b/www/command-line-options.html
index d402774c3..bb7c9071d 100644
--- a/www/command-line-options.html
+++ b/www/command-line-options.html
@@ -8034,4 +8034,3 @@ but with strict boolean masking. </p>
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 10:56 -->
\ No newline at end of file
diff --git a/www/command-line-processing.html b/www/command-line-processing.html
index 19f7bad3a..d3d713b63 100644
--- a/www/command-line-processing.html
+++ b/www/command-line-processing.html
@@ -685,4 +685,3 @@ convert -define stream:buffer-size=0 logo: gif:- | display gif:-
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 10:57 -->
\ No newline at end of file
diff --git a/www/command-line-tools.html b/www/command-line-tools.html
index 8eb3e01b9..0b92393de 100644
--- a/www/command-line-tools.html
+++ b/www/command-line-tools.html
@@ -100,4 +100,3 @@
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 10:58 -->
\ No newline at end of file
diff --git a/www/compare.html b/www/compare.html
index b3b084051..0e809a436 100644
--- a/www/compare.html
+++ b/www/compare.html
@@ -365,4 +365,3 @@ transparent, extract, background, or shape the alpha channel</td>
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:21 -->
\ No newline at end of file
diff --git a/www/compose.html b/www/compose.html
index 8129b7c5c..e67830922 100644
--- a/www/compose.html
+++ b/www/compose.html
@@ -644,4 +644,3 @@ href="command-line-options.html#list">-list compose</a>.</p>
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:57 -->
\ No newline at end of file
diff --git a/www/composite.html b/www/composite.html
index caa3f4c89..5b3cdcfd3 100644
--- a/www/composite.html
+++ b/www/composite.html
@@ -526,4 +526,3 @@ transparent, extract, background, or shape the alpha channel</td>
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:15 -->
\ No newline at end of file
diff --git a/www/conjure.html b/www/conjure.html
index f52a834ec..9dfba1937 100644
--- a/www/conjure.html
+++ b/www/conjure.html
@@ -1138,4 +1138,3 @@ fill="color name", channel="All, Default, Alpha, Black, Blue, CMYK, Cyan, Gray,
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:30 -->
\ No newline at end of file
diff --git a/www/connected-components.html b/www/connected-components.html
index 89ee0eb67..125d20037 100644
--- a/www/connected-components.html
+++ b/www/connected-components.html
@@ -121,4 +121,3 @@ Objects (id: bounding-box centroid area mean-color):
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 13:52 -->
\ No newline at end of file
diff --git a/www/convert.html b/www/convert.html
index 831022775..0362e8fce 100644
--- a/www/convert.html
+++ b/www/convert.html
@@ -1322,4 +1322,3 @@ transparent, extract, background, or shape the alpha channel</td>
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 10:56 -->
\ No newline at end of file
diff --git a/www/display.html b/www/display.html
index 4776941d6..bb468af14 100644
--- a/www/display.html
+++ b/www/display.html
@@ -569,4 +569,3 @@ transparent, extract, background, or shape the alpha channel</td>
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:18 -->
\ No newline at end of file
diff --git a/www/distribute-pixel-cache.html b/www/distribute-pixel-cache.html
index 2924eede8..7f0f3ad64 100644
--- a/www/distribute-pixel-cache.html
+++ b/www/distribute-pixel-cache.html
@@ -89,4 +89,3 @@ convert -limit memory 1GiB -limit map 2GiB -limit disk 4GiB \
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 13:03 -->
\ No newline at end of file
diff --git a/www/download.html b/www/download.html
index e44e25046..2b69a955e 100644
--- a/www/download.html
+++ b/www/download.html
@@ -115,4 +115,3 @@
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 10:56 -->
\ No newline at end of file
diff --git a/www/escape.html b/www/escape.html
index b45da293e..a188fe50f 100644
--- a/www/escape.html
+++ b/www/escape.html
@@ -887,4 +887,3 @@ Post ObjectData Descriptor Record
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 10:58 -->
\ No newline at end of file
diff --git a/www/examples.html b/www/examples.html
index 946f6cf06..43d92408a 100644
--- a/www/examples.html
+++ b/www/examples.html
@@ -80,4 +80,3 @@
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:04 -->
\ No newline at end of file
diff --git a/www/exception.html b/www/exception.html
index 88d124550..af98167de 100644
--- a/www/exception.html
+++ b/www/exception.html
@@ -269,4 +269,3 @@
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 16:10 -->
\ No newline at end of file
diff --git a/www/export.html b/www/export.html
index 1749565cf..dd4e8a30d 100644
--- a/www/export.html
+++ b/www/export.html
@@ -80,4 +80,3 @@
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 16:23 -->
\ No newline at end of file
diff --git a/www/formats.html b/www/formats.html
index c0f53bbfb..27212b2f1 100644
--- a/www/formats.html
+++ b/www/formats.html
@@ -1932,4 +1932,3 @@ convert \( -size 15x15 canvas:black canvas:white -append \) \
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:31 -->
\ No newline at end of file
diff --git a/www/fx.html b/www/fx.html
index 9005339df..d5d5f0c41 100644
--- a/www/fx.html
+++ b/www/fx.html
@@ -420,4 +420,3 @@ convert rose: -duplicate 29 -virtual-pixel Gray -distort SRT '%[fx:360.0*t/n]' \
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:03 -->
\ No newline at end of file
diff --git a/www/gradient.html b/www/gradient.html
index 04271ac87..2f0681a4d 100644
--- a/www/gradient.html
+++ b/www/gradient.html
@@ -241,4 +241,3 @@ convert -size 256x256 -define gradient:radii=128,64 -define gradient:angle=45 ra
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:33 -->
\ No newline at end of file
diff --git a/www/high-dynamic-range.html b/www/high-dynamic-range.html
index 3b6fcdc1a..b5a9e2385 100644
--- a/www/high-dynamic-range.html
+++ b/www/high-dynamic-range.html
@@ -107,4 +107,3 @@ Features: HDRI
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 12:53 -->
\ No newline at end of file
diff --git a/www/history.html b/www/history.html
index f487703ae..f4377b4b0 100644
--- a/www/history.html
+++ b/www/history.html
@@ -97,4 +97,3 @@ I swear by my life and my love of it that I will never live for the sake of anot
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 16:09 -->
\ No newline at end of file
diff --git a/www/identify.html b/www/identify.html
index 54bbdf372..0356a6f5e 100644
--- a/www/identify.html
+++ b/www/identify.html
@@ -433,4 +433,3 @@ transparent, extract, background, or shape the alpha channel</td>
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:03 -->
\ No newline at end of file
diff --git a/www/import.html b/www/import.html
index 9946a0db1..19b77e9f1 100644
--- a/www/import.html
+++ b/www/import.html
@@ -427,4 +427,3 @@ import -window root screen.ps
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:57 -->
\ No newline at end of file
diff --git a/www/index.html b/www/index.html
index 44f97a607..13f4faaf8 100644
--- a/www/index.html
+++ b/www/index.html
@@ -239,4 +239,3 @@ Use ImageMagick<sup><a href="http://tarr.uspto.gov/servlet/tarr?regser=serial&am
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 10:56 -->
\ No newline at end of file
diff --git a/www/install-source.html b/www/install-source.html
index 74999f777..d9418676c 100644
--- a/www/install-source.html
+++ b/www/install-source.html
@@ -162,4 +162,3 @@ validate
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:00 -->
\ No newline at end of file
diff --git a/www/jp2.html b/www/jp2.html
index 33bd9bc0e..5f879d959 100644
--- a/www/jp2.html
+++ b/www/jp2.html
@@ -164,4 +164,3 @@ convert wizard.png -resize 4096x2160! -depth 12 wizard.jp2
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:38 -->
\ No newline at end of file
diff --git a/www/license.html b/www/license.html
index 5cc7ea0a9..c22a2f01a 100644
--- a/www/license.html
+++ b/www/license.html
@@ -203,4 +203,3 @@
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:07 -->
\ No newline at end of file
diff --git a/www/links.html b/www/links.html
index 87ce68b7d..b36b34ebc 100644
--- a/www/links.html
+++ b/www/links.html
@@ -163,4 +163,3 @@
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:37 -->
\ No newline at end of file
diff --git a/www/magick++.html b/www/magick++.html
index c9728f512..f6fba6494 100644
--- a/www/magick++.html
+++ b/www/magick++.html
@@ -187,4 +187,3 @@ int main(int argc,char **argv)
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 12:07 -->
\ No newline at end of file
diff --git a/www/magick-core.html b/www/magick-core.html
index 1918ec661..cacec6bf4 100644
--- a/www/magick-core.html
+++ b/www/magick-core.html
@@ -337,4 +337,3 @@ int main(int argc,char **argv)
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:04 -->
\ No newline at end of file
diff --git a/www/magick-script.html b/www/magick-script.html
index 4f2032666..3fe15ce79 100644
--- a/www/magick-script.html
+++ b/www/magick-script.html
@@ -1282,4 +1282,3 @@ transparent, extract, background, or shape the alpha channel</td>
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:31 -->
\ No newline at end of file
diff --git a/www/magick-vector-graphics.html b/www/magick-vector-graphics.html
index f97fac07d..8a7dfb283 100644
--- a/www/magick-vector-graphics.html
+++ b/www/magick-vector-graphics.html
@@ -881,4 +881,3 @@ round</pre></td>
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:26 -->
\ No newline at end of file
diff --git a/www/magick-wand.html b/www/magick-wand.html
index 616320eda..4d08d8133 100644
--- a/www/magick-wand.html
+++ b/www/magick-wand.html
@@ -383,4 +383,3 @@ int main(int argc,char **argv)
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 12:34 -->
\ No newline at end of file
diff --git a/www/magick.html b/www/magick.html
index d8c24b304..3990f7633 100644
--- a/www/magick.html
+++ b/www/magick.html
@@ -1324,4 +1324,3 @@ transparent, extract, background, or shape the alpha channel</td>
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:11 -->
\ No newline at end of file
diff --git a/www/miff.html b/www/miff.html
index 83107f7d2..b7392ae6f 100644
--- a/www/miff.html
+++ b/www/miff.html
@@ -245,4 +245,3 @@ or fewer colors in the image, each byte of image data contains an index value. I
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 14:58 -->
\ No newline at end of file
diff --git a/www/mogrify.html b/www/mogrify.html
index 7377006fa..96de2cc94 100644
--- a/www/mogrify.html
+++ b/www/mogrify.html
@@ -1297,4 +1297,3 @@ transparent, extract, background, or shape the alpha channel</td>
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:01 -->
\ No newline at end of file
diff --git a/www/montage.html b/www/montage.html
index d0eab324b..4f6b0e396 100644
--- a/www/montage.html
+++ b/www/montage.html
@@ -638,4 +638,3 @@ transparent, extract, background, or shape the alpha channel</td>
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:41 -->
\ No newline at end of file
diff --git a/www/motion-picture.html b/www/motion-picture.html
index ba8a90943..9adef4d51 100644
--- a/www/motion-picture.html
+++ b/www/motion-picture.html
@@ -194,4 +194,3 @@ convert bluebells.dpx -define dpx:television.time.code=10:00:02:15 bluebells-001
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 12:05 -->
\ No newline at end of file
diff --git a/www/opencl.html b/www/opencl.html
index 66451fc00..85742647d 100644
--- a/www/opencl.html
+++ b/www/opencl.html
@@ -122,4 +122,3 @@ InitImageMagickOpenCL(MAGICK_OPENCL_DEVICE_SELECT_USER, (void*)(myDevices+2), NU
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 12:36 -->
\ No newline at end of file
diff --git a/www/openmp.html b/www/openmp.html
index 559ad4850..3116b0ab5 100644
--- a/www/openmp.html
+++ b/www/openmp.html
@@ -101,4 +101,3 @@ Performance[8]: 40i 4.831ips 0.872e 15.680u 0:02.070
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 12:43 -->
\ No newline at end of file
diff --git a/www/perl-magick.html b/www/perl-magick.html
index bc836a5ed..c3193faf3 100644
--- a/www/perl-magick.html
+++ b/www/perl-magick.html
@@ -2611,4 +2611,3 @@ Image::Magick-&gt;QuantumDepth
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:31 -->
\ No newline at end of file
diff --git a/www/porting.html b/www/porting.html
index 66f3fbb97..18ef5c6af 100644
--- a/www/porting.html
+++ b/www/porting.html
@@ -662,4 +662,3 @@ example "+annotate", "+resize", "+clut", and "+draw" .</p>
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:31 -->
\ No newline at end of file
diff --git a/www/quantize.html b/www/quantize.html
index 0528fa2b0..0d5f8010b 100644
--- a/www/quantize.html
+++ b/www/quantize.html
@@ -202,4 +202,3 @@ while number of nodes with (n2 &gt; 0) &gt; required maximum number of colors
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 16:17 -->
\ No newline at end of file
diff --git a/www/resources.html b/www/resources.html
index 267cec87a..78789f18a 100644
--- a/www/resources.html
+++ b/www/resources.html
@@ -383,4 +383,3 @@ $MAGICK_FONT_PATH
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:08 -->
\ No newline at end of file
diff --git a/www/security-policy.html b/www/security-policy.html
index caf81537f..a2ba47753 100644
--- a/www/security-policy.html
+++ b/www/security-policy.html
@@ -184,4 +184,3 @@ Path: [built-in]
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:49 -->
\ No newline at end of file
diff --git a/www/sitemap.html b/www/sitemap.html
index 1101a9ca0..f9f68d6b6 100644
--- a/www/sitemap.html
+++ b/www/sitemap.html
@@ -235,4 +235,3 @@
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 10:58 -->
\ No newline at end of file
diff --git a/www/stream.html b/www/stream.html
index 99c36ce73..90d7c0c7b 100644
--- a/www/stream.html
+++ b/www/stream.html
@@ -272,4 +272,3 @@ stream -map i -storage-type double 'image.tif[100x100+30+40]' gray.raw
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 11:58 -->
\ No newline at end of file
diff --git a/www/support.html b/www/support.html
index 5bc079042..0e72efe4f 100644
--- a/www/support.html
+++ b/www/support.html
@@ -155,4 +155,3 @@ USA
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 12:40 -->
\ No newline at end of file
diff --git a/www/webp.html b/www/webp.html
index db08d74a6..7a7ff2d3f 100644
--- a/www/webp.html
+++ b/www/webp.html
@@ -182,4 +182,3 @@ convert wizard.png -quality 50 -define webp:lossless=true wizard.webp
 </div>
 </body>
 </html>
-<!-- Magick Cache 24th December 2016 15:16 -->
\ No newline at end of file
